packages/splitter/src/useSplitter.ts

```diff
@@ -190,9 +247,9 @@ export function useSplitter({
       composeEventHandlers(props.onMouseUp, props.onMouseLeave, (event: MouseEvent) => {
         event.preventDefault();
         // must remove global events on transaction finish
-        environment.removeEventListener('mouseup', onMouseLeaveOrUp);
-        environment.body.removeEventListener('mouseleave', onMouseLeaveOrUp);
-        environment.removeEventListener('mousemove', onMouseMove);
+        environment.document.removeEventListener('mouseup', onMouseLeaveOrUp);
+        environment.document.body.removeEventListener('mouseleave', onMouseLeaveOrUp);
```

Comment:
I'm curious, why do we need to target the document body with `mouseleave` event? Why not just the document?

Reply:
`document.body` will fire `mouseleave` in all browsers. `document` will fire `mouseleave` on all browsers except IE11.